test/test_config.ts

```diff
@@ -127,8 +127,8 @@ describe('config types', function () {
                     channel: 'lonewolf-games',
                     channelId: 'C0SD3SCAH',
                     clock: {
-                        initial: 30,
-                        increment: 30,
+                        initial: 1,
```

Comment:
Yeah, all of these changes should be ignored. this config is setup to test on the live slack

Reply:
You can feel free to make and commit another config that you use for your slack, if you'd like